CMSIS/DoxyGen/Driver/src/Driver_SAI.c

```diff
@@ -281,6 +281,9 @@ The <b>MCLK pin</b> setting requested with the function \ref ARM_SAI_Control is
 \def ARM_SAI_ERROR_MCLK_PRESCALER       
 The <b>MCLK prescaler</b> requested with the function \ref ARM_SAI_Control is not supported.
 
+\def ARM_SAI_ERROR_FRAME_LENGTH
+The <b>frame length</b> requested with the function \ref ARM_SAI_Control is not supported.
+
 \def ARM_SAI_ERROR_FRAME_LENGHT
```

Comment:
The @deprecated needs to go in here in order to appear in the generated documentation.

Reply:
Sorry, all is okay. I'll pick it up.